### PR TITLE
[FEATURE] Apply Singleton pattern to EuTxManager

### PR DESCRIPTION
**Pull request by SeonJK**
- relates to : #260 

- Adapt bill pugh solution

### CODE REVIEW

**Review by designe — COMMENTED** [review]
Hi @SeonJK!
Thanks for your singleton suggestion :)

I made a comment. please check it out.

File: euphony/src/main/java/co/euphony/tx/EuTxManager.kt [hunk 1]
```diff
@@ -29,6 +29,14 @@ class EuTxManager {
             txCore.code = data
         }
 
+    companion object {
+        val instance: EuTxManager by lazy { Holder.INSTANCE }
+    }
```
[the remark this answers is not in the document]
Comment (SeonJK) [reply euphony/src/main/java/co/euphony/tx/EuTxManager.kt]: Thanks for request! It's much easier to use . I adapted it!!

**Review by designe — COMMENTED** [review]
+ And here are last modifications! 👍 
- https://github.com/euphony-io/euphony/blob/master/README.md
- https://github.com/euphony-io/euphony/blob/master/GETTING_STARTED.md
- https://github.com/euphony-io/euphony/blob/master/GETTING_STARTED_KR.md

**Comment by SeonJK** [comment]
> * And here are last modifications! 👍
> 
> * https://github.com/euphony-io/euphony/blob/master/README.md
> * https://github.com/euphony-io/euphony/blob/master/GETTING_STARTED.md
> * https://github.com/euphony-io/euphony/blob/master/GETTING_STARTED_KR.md

Oh, I forgot these.... I modify them!!

**Review by designe — APPROVED** [review]

**Pull request merged**